server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -203,6 +260,13 @@ public ClusterMetadataManifest(StreamInput in) throws IOException {
         this.indices = Collections.unmodifiableList(in.readList(UploadedIndexMetadata::new));
         this.previousClusterUUID = in.readString();
         this.clusterUUIDCommitted = in.readBoolean();
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
```

Comment:
lets make sure we update this during backport to 2.x

Reply:
Yes we will update this when backporting to 2.x